src/app/login/login.component.ts

```diff
@@ -6,6 +6,7 @@ import { Component, OnInit } from '@angular/core';
   styleUrls: ['./login.component.css']
 })
 export class LoginComponent implements OnInit {
+	model:any = {name:'abc' , pwd:'12345'};
```

Comment:
rename model name and declare it as empty object {}
also the keys name should be full like username and password